### ISSUE TITLE
Remote Access

### ISSUE THREAD

**Issue by sburke781** [andyvorld/LGSTrayBattery#22]
I was hoping to use this app to access the battery levels of my mouse and keyboard from my home automation hub (Hubitat).  I can't seen to get it to access the devices page to get the listing.  I have tried opening up the port being used through the Windows Firewall policies, but still cannot get it to work.  Localhost calls work locally on the host laptop, but not from outside, or even by using the IP address in a local browser.

Any thoughts / tips?

**Comment by andyvorld** [comment]
Yeah, I am trying to have a look at making it configurable from the Httpserver.ini bindings so that it can still bind to localhost by default, but allows you to open up for external access if needed.

**Comment by sburke781** [comment]
I might have logged this under the wrong repo, I use the G-Hub version...  Yes looks like I have, my apologies....

**Comment by sburke781** [comment]
Well, locally, still needing to work on remote access.  Getting a ClientProtocolException when calling from my hub.  Have tried port 12321 and 8081, but neither work remotely at this stage.  Have added Firewall rule to open the port up, but still no luck....  Will keep looking at it.  Thanks again, Simon

**Comment by sburke781** [comment]
I have seen people talk about the error relating to no headers being included.
